distribution/docs/src/main/resources/_contents/_resources/content-directory-monitor-contents.adoc

```diff
@@ -69,10 +69,9 @@ For example, to set the contact email for all metacards, add the custom mapping:
 Each mapping sets the value of a single metacard attribute. To set the value of an additional attribute, select the "plus"
  icon in the UI. This creates an empty line for the entry.
 
-To set multi-valued attributes, use a separate override for each value. For example, to add the keywords _PPI_ and _radiology_ to each metacard, add the custom attribute mappings:
+To set multi-valued attributes, use a comma separated list to list all values for a specified attribute. For example, to add the keywords _PPI_ and _radiology_ to each metacard, add the custom attribute mappings:
```

Comment:
❓ should we clarify the example a little more "add the custom attribute **topic.keyword** mappings"? Something like that? Maybe it is more clear in full context but just clarifying which part is the attribute.